api/build.gradle

```diff
@@ -428,34 +433,38 @@ dependencies {
 
   compile "org.json:json:20160810"
 
-  compile("org.springframework.boot:spring-boot-starter-actuator:$project.ext.SPRING_BOOT_VERSION") {
+  compile("org.apache.tomcat:tomcat-jdbc:9.0.10")
+  compile("org.springframework.boot:spring-boot-starter-tomcat")
+  compile("org.springframework.boot:spring-boot-starter-jdbc")
+  compile("org.springframework.boot:spring-boot-starter-actuator") {
```

Comment:
Do we need this one?

Reply:
That one was auto imported by some tool I remembeted. Let me try to remove this..